distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -37,8 +39,6 @@ log4j2.rootLogger.appenderRef.osgi-all.ref = osgi-all
 log4j2.logger.securityLogger.name = securityLogger
 log4j2.logger.securityLogger.level = INFO
 log4j2.logger.securityLogger.additivity = false
-# TODO DDF-3073 (see note below)
-#log4j2.logger.securityLogger.appenderRef.securityFailover.ref = securityFailover
 log4j2.logger.securityLogger.appenderRef.securityMain.ref = securityMain
```

Comment:
This line should be removed because `securityMain` is the primary appender of the `securityFailover` Failover.